test/constructor.js

```diff
@@ -6,22 +6,7 @@ var Nemo = require('nemo'),
   nemo = {};
 
 describe('nemo-view @constructor@', function () {
-  it('should do ? with malformed JSON file(s)', function (done) {
```

Comment:
So the current behavior is to kill the setup with an error. I think we still should, and thus this test case is still valid. But we should do so with a better message.